starboard/testing/run_all_unittests_starboard.cc

```diff
@@ -28,7 +28,7 @@ int LaunchUnitTests(int argc, char** argv) {
 }  // namespace
 
 // For the Starboard OS define SbEventHandle as the entry point
-SB_EXPORT STARBOARD_WRAP_SIMPLE_MAIN(LaunchUnitTests);
+SB_EXPORT STARBOARD_WRAP_SIMPLE_MAIN(LaunchUnitTests)
```

Comment:
Same comment as other macros: I think these macros should be written to accept trailing semicolons.